### PR TITLE
Improve documentation for weather_forecast script

### PR DESCRIPTION
**Pull request by snehampatil**
Updated the readme
Added the steps for installation and execution.

### CODE REVIEW

File: scripts/weather_forecast/readme.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
-a script to get weather forecast. 
-requirements:
-1: geopy
-usage :
-python forecast.py city_name
+# weather_forecast
+## About
+A Python script to get weather forecast for a particular city. 
+## Setup
+**Step1:** Install Python3 [Link](https://github.com/geopy/geo)
```
Comment (OBITORASU) [inline scripts/weather_forecast/readme.md:5]: Fix this link

**Review by OBITORASU — CHANGES_REQUESTED** [review]
I will consider this since this is some proper documentation. Left a review.

File: scripts/weather_forecast/readme.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+```
+pip install geopy
+```
+**Step3:** Clone this repository
+```
+https://github.com/GDSC-RCCIIT/General-Purpose-Scripts.git
+```
+**Step4:** Change the directory
+```
+cd General-Purpose-Scripts\scripts\weather_forecast
```
Comment (OBITORASU) [inline scripts/weather_forecast/readme.md:16]: I would prefer if you use forward slash instead of backslash on this one.

Comment (snehampatil) [reply scripts/weather_forecast/readme.md]: I changed it

File: scripts/weather_forecast/readme.md (hunk 1 above)
Comment (snehampatil) [reply scripts/weather_forecast/readme.md]: Fixed it.

**Pull request merged**